### PR TITLE
Required contact info

### PR DESCRIPTION
**Pull request by Omarkojak**
For issue #162

### CODE REVIEW

**Comment by Omarkojak** [comment]
@AhmadElsagheer

File: app/Http/Controllers/Event/EventController.php [hunk 1]
```diff
@@ -137,8 +137,10 @@ public function unfollow($id)
 
 	public function register($id)
 	{
-		$this->eventService->register($id, Auth::user());
-		return redirect()->action('Event\EventController@show', [$id]);
+		$validator = $this->eventService->register($id, Auth::user());
+		if($validator->passes())
+			return redirect()->action('Event\EventController@show', [$id]);
+		return redirect()->action('Event\EventController@show', [$id])->withErrors($validator)->withInput();
```
Comment (AhmadElsagheer) [inline app/Http/Controllers/Event/EventController.php]: why withInput?

**Comment by Omarkojak** [comment]
sorry mkntsh a3rf enha bt3ml kda ana knt fakr enha dih eli btl3 errors elvalidator @AhmadElsagheer

**Comment by Omarkojak** [comment]
sorry aflto bel3.alat

**Pull request merged**